src/pages/events.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+    <common-page-block>
+        <template #title>VatSIM Events</template>
+
+        <common-event-card
+            v-for="event in data?.events"
+            :key="event.id"
+            class="common-event"
+            :event
+        />
+    </common-page-block>
+</template>
+
+<script setup lang="ts">
+import CommonPageBlock from '~/components/common/blocks/CommonPageBlock.vue';
+import type { VatsimDivision, VatsimEvent, VatsimSubDivision } from '~/types/data/vatsim';
+import CommonEventCard from '~/components/common/vatsim/CommonEventCard.vue';
+
+const { data } = await useAsyncData('events', async () => {
+    return $fetch<{ events: VatsimEvent[]; divisions: VatsimDivision[]; subDivisions: VatsimSubDivision[] }>('/api/data/vatsim/events');
```

Comment:
Let's make this a type/interface somewhere

Reply:
Done.